src/VisualStudio/Core/Def/Implementation/ProjectSystem/FileChangeWatcher.cs

```diff
@@ -76,20 +77,49 @@ public void WaitForQueue_TestOnly()
             queue.Wait();
         }
 
-        public IContext CreateContext()
-            => new Context(this, null);
+        public IContext CreateContext(params WatchedDirectory[] watchedDirectories)
+        {
+            return new Context(this, watchedDirectories.ToImmutableArray());
```

Comment:
Keep as an expression-bodied-method?

Reply:
Meh. I can't say I care for them much beyond trivial accessors anyways. :-)